src/sessions/session-manager.ts

```diff
@@ -68,6 +79,19 @@ export class SessionManager {
     return session;
   }
 
+  getAllActiveSessions (): IJingleSession[] {
+    const sessions = [];
+
+    Object.keys(this.jingle.sessions).forEach(key => {
```

Comment:
NIT: since you don't really need the key here, you could just do `Object.values().filter(session => session.active)`